.github/workflows/pr-validate.yml

```diff
@@ -57,7 +57,7 @@ jobs:
 
   build-dotnet:
     name: ".NET"
-    uses: trinsic-id/okapi/.github/workflows/build-dotnet.yml@main
+    uses: trinsic-id/okapi/.github/workflows/build-dotnet.yml@TM/net-targets
```

Comment:
not a sustainable path unless this branch is always available.

Reply:
No, but required to pass PR, since the workflow itself has changes. Will revert once this is merged